task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -12,33 +44,9 @@ static void Main(string[] args)
             if (arrayLength <= 5)
             {
                 int[] numberArray = new int[arrayLength];
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    Console.WriteLine($"Enter an array element № {i}");
-                    numberArray[i] = Convert.ToInt32(Console.ReadLine());
-                }
 
-                bool isPrimeNumber;
-                for (int i = 0; i < arrayLength; i++)
-                {
-                    isPrimeNumber = true;
-                    for (int j = 2; j < numberArray[i]; j++)
-                    {
-                        if (numberArray[i] % j == 0)
-                        {
-                            isPrimeNumber = false;
-                            break;
-                        }
-                    }
-                    if (isPrimeNumber)
-                    {
-                        Console.WriteLine($"{numberArray[i]} is prime number");
-                    }
-                    else
-                    {
-                        Console.WriteLine($"{numberArray[i]} is not prime number");
-                    }
-                }
+                InputOfArrayElements(numberArray, arrayLength);
```

Comment:
don't forget to add variable as this method CreateNumbersArray will return the array.

Reply:
done